ingestion/strip_data.py

```diff
@@ -0,0 +1,136 @@
+"""This script strips the raw data to only include the columns we need for the project."""
+
+import pandas as pd
+import logging
+
+def austin():
+    """
+    Extracts needed columns from the Austin data and saves it as a csv file
+    :return austin_stripped: the stripped data as a pandas dataframe
+    """
+    austin_raw = pd.read_csv("./raw_data/austin.csv")
+    austin_stripped = austin_raw[austin_raw.contractor_trade == "Electrical Contractor"]
+    austin_stripped = austin_raw[["issue_date", "location", "contractor_company_name"]]
+    latitude = []
+    longitude = []
+    for i in range(len(austin_stripped) - 1):
+        # {'latitude': '30.29406429', 'longitude': '-97.69323996', 'human_address': '{""address"": """", ""city"": """", ""state"": """", ""zip"": """"}'}
+        location_object = austin_stripped["location"][i]
+        if type(location_object) != str:
+            latitude.append("nan")
+            longitude.append("nan")
+            continue
+        latitude_temp, longitude_temp = location_object.split(",")[:2]
+        latitude.append(latitude_temp.split("'")[3])
+        longitude.append(longitude_temp.split("'")[3])
+    austin_stripped["latitude"] = pd.Series(latitude)
+    austin_stripped["longitude"] = pd.Series(longitude)
+    austin_stripped = austin_stripped.dropna(subset=["contractor_company_name"])
+    austin_stripped.drop(columns=["location"], inplace=True)
+    austin_stripped["issue_date"] = austin_stripped["issue_date"].apply(lambda x: x.split("T")[0])
+    austin_stripped.to_csv("./stripped_data/austin.csv")
+    logging.info("Saved austin.csv")
+    return austin_stripped
+
+def new_york():
+    """
+    Extracts needed columns from the New York data and saves it as a csv file
+    :return new_york_stripped: the stripped data as a pandas dataframe
+    """
+    new_york_raw = pd.read_csv("./raw_data/new_york.csv")
+    new_york_stripped = new_york_raw[["job_start_date","firm_name", 'gis_latitude', 'gis_longitude']]
+    new_york_stripped = new_york_stripped.dropna(subset=["firm_name"])
+    new_york_stripped["job_start_date"] = new_york_stripped["job_start_date"].apply(lambda x: x.split("T")[0])
+    new_york_stripped.to_csv("./stripped_data/new_york.csv")
+    logging.info("Saved new_york.csv")
+    return new_york_stripped
+
+def chicago():
+    """
+    Extracts needed columns from the Chicago data and saves it as a csv file
+    :return: chicago_stripped: the stripped data as a pandas dataframe
+    """
+    chicago_raw = pd.read_csv("./raw_data/chicago.csv")
+    chicago_stripped = chicago_raw[chicago_raw.contact_1_type == "CONTRACTOR-ELECTRICAL"]
+    chicago_stripped = chicago_stripped[["issue_date", "contact_1_name", "latitude", "longitude"]]
```

Comment:
You are only checking for contractors in the contact_1 columns, but it looks like there are columns for contact_2, contact_3, etc. Can you incorporate those as well